lib/config/fileconf.go

```diff
@@ -437,6 +437,15 @@ func (conf *FileConfig) CheckAndSetDefaults() error {
 		}
 	}
 
+	for i := 0; i < len(conf.Databases.AzureMatchers); i++ {
+		matcher := &conf.Databases.AzureMatchers[i]
+		if len(matcher.Subscriptions) == 0 || apiutils.SliceContainsStr(matcher.Subscriptions, types.Wildcard) {
+			matcher.Subscriptions = []string{types.Wildcard}
+		}
+		if len(matcher.ResourceGroups) == 0 || apiutils.SliceContainsStr(matcher.ResourceGroups, types.Wildcard) {
+			matcher.ResourceGroups = []string{types.Wildcard}
+		}
+	}
```

Comment:
This probably can be moved to applyDatabasesConfig to keep this kind of logic in one place.

Reply:
Ok, yeah I wasn't sure where to put this, but this place looked like the entry point for any config. I'll move it